app/Models/RockTheVoteReport.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Chompy\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class RockTheVoteReport extends Model
+{
+    /**
+     * We use the externally created Rock the Vote ID as our primary key.
+     */
+    public $incrementing = false;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'id',
+        'status',
+        'since',
+        'before',
```

Comment:
Since these are dates, do we want to add them to the [`$dates`](https://laravel.com/docs/7.x/eloquent-mutators#date-mutators) property on the model?